data-sources/gitlab/config.yml

```diff
@@ -0,0 +1,22 @@
+id: otel-gitlab-integration
+displayName: Gitlab
+description: |
+  GitLab is a popular single application for the entire DevOps lifecycle, fulfilling all the fundamentals for CI/CD in one environment.
+icon: logo.png
+install:
+  primary:
+    nerdlet:
+      nerdletId: nr1-install-newrelic.quickstart-installation-plan
+      nerdletState:
+        quickstartId: 980ab07a-3483-4b44-bbac-ed9c0a9a41a1
```

Comment:
This `quickstartId` points to the jenkins quickstart. I'm not seeing a clear link between gitlab and jenkins with this integration. 

The jenkins data source has a very specific integration on the backend, which leads to my confusion.

Would you mind explaining this? I may totally be missing something!

Reply:
Resolved